src/Components/PressableRequirement/styles.ts

```diff
@@ -8,10 +8,11 @@ export default StyleSheet.create({
     paddingTop: 20,
     columnGap: 20,
     paddingHorizontal: '6%',
```

Comment:
paddingTop is overwritten by paddingVertical, so you can remove this! Also, for consistency sake, change `paddingHorizontal: '6%'` to `paddingHorizontal: 25`

_Sent from Smart Toilet_
Best,
Philip